views/problem_set.ejs

```diff
@@ -37,18 +37,22 @@
                     </tr>
                 </thead>
                 <tbody>
+                 <% for(var i=0; i<db.length; i++) {%>
                     <tr>
-                        <td><a href="/problem/4">4</a></td> 
-                        <td><a href="/problem/4">Dummy Name</a></td>
-                        <td>math, brute force</td>
-                        <td>easy</td>
-                    </tr>
-                    <tr>
-                        <td><a href="/problem/6">6</a></td> 
-                        <td><a href="/problem/6">Again Dummy Name</a></td>
-                        <td>math, brute force</td>
-                        <td>easy</td>
+                      <td>
+                        <%= db[i]._id %>
```

Comment:
Here also, `_id` to `qID`. And add a link to this text to http://localhost:3000/problem/:qID